site/src/components/Resources/AgentStatus.tsx

```diff
@@ -50,7 +56,52 @@ const ConnectingStatus: React.FC = () => {
   )
 }
 
-export const AgentStatus: React.FC<{ agent: WorkspaceAgent }> = ({ agent }) => {
+const TimeoutStatus: React.FC<{
+  agent: WorkspaceAgent
+  workspace: Workspace
+}> = ({ agent, workspace }) => {
+  const { t } = useTranslation("agent")
+  const styles = useStyles()
+  const anchorRef = useRef<SVGSVGElement>(null)
+  const [isOpen, setIsOpen] = useState(false)
+  const id = isOpen ? "timeout-popover" : undefined
+  const troubleshootLink =
+    agent.troubleshooting_url ?? `/templates/${workspace.template_name}#readme`
```

Comment:
The `#readme` didn't cause my browser (Safari) to scroll the readme into view, should that happen?

Reply:
Fixed!